cdci_data_analysis/analysis/renku_helper.py

```diff
@@ -219,6 +282,17 @@ def generate_nb_hash(nb_obj):
     return notebook_hash
 
 
+def create_renku_ini_config_obj(repo, default_url_file_name):
+    repo_dir = repo.working_dir
+
+    renku_ini_path = os.path.join(repo_dir, '.renku', 'renku.ini')
+
+    renku_config = ConfigParser()
+    renku_config.read(renku_ini_path)
+    renku_config['renku "interactive"']['default_url'] = f'/lab/tree/{default_url_file_name}'
```

Comment:
just wondering, as we now have unique branch, is it possible to set the file name (and the default url name) to something simple, like `api.ipynb`? It looks a bit less nice when it's shown like it is now.

Reply:
I agree it'd look much cleaner, but by doing so the content of the renku ini file will always be the same, and so the resulting hash will. I suppose is no gonna be a major issue, but we should keep it in mind.

Reply:
hopefully it will be ok

Reply:
It is now fixed to `api_copde.ipynb`